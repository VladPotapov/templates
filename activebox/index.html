<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>ActiveBox</title>
    <link rel="stylesheet" href="css/style.css">
</head>
<body>
    <header class="header">
        <div class="container">
            <div class="header__inner">
                <div class="header__logo">
                    <img src="img/logo.png" alt="">
                </div>
                <!-- /.header__logo -->
                <nav class="nav">
                    <a href="" class="nav__link">feature</a>
                    <a href="" class="nav__link">works</a>
                    <a href="" class="nav__link">our time</a>
                    <a href="" class="nav__link">testimonials</a>
                    <a href="" class="nav__link">download</a>
                </nav>
            </div>
            <!-- /.header__inner -->
        </div>
        <!-- /.container -->
    </header>
    <div class="intro">
        <div class="container">
            <div class="intro__inner">
                <h1 class="intro__title">Your Favorite One Page <br> Multi Purpose Template</h1>
                <h2 class="intro__subtitle">
                    Lorem ipsum dolor sit amet, consectetur adipiscing elit. Praesent commodo cursus magna vel scelerisque nisl consectetur et.
                </h2>
            </div>
            <!-- /.intro__inner -->
        </div>
        <!-- /.container -->
    </div>
    <!-- /.intro -->
    <script src="js/script.js"></script>
</body>
</html>