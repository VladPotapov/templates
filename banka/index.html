<!DOCTYPE html>
<html lang="ru" class="no-js">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <meta http-equiv="X-UA-Compatible" content="ie=edge">
    <link href="https://fonts.googleapis.com/css?family=Roboto:400,700&display=swap" rel="stylesheet">
    <link rel="stylesheet" href="css/style.css">
    <title>Храните деньги в трёхлитровой банке - Пенькоф</title>
</head>
<body>
    <header class="header">
        <div class="container">
            <div class="header-wrap">
                <div class="logo">Пенькоф</div>
                <nav class="nav">
                    <a href="" class="nav__link">Банки</a>
                    <a href="" class="nav__link">Чулки</a>
                    <a href="" class="nav__link">Под подушкой</a>
                    <a href="" class="nav__link">Страховка</a>
                    <a href="" class="nav__link">Кэшбек</a>
                </nav>
                <a href="" class="login">Войти</a>
            </div>
            <!-- /.header-wrap -->
        </div>
        <!-- /.container -->
    </header>

    <section>
        <div class="container">
            <div class="tabs">
                <h1 class="tabs__logo">Penkoff Blank</h1>
                <!-- /.tabs__logo -->
                <ul class="tabs-menu">
                    <li class="tabs__item">Обзор</li>
                    <li class="tabs__item">Примеры</li>
                    <li class="tabs__item">Документы</li>
                    <li class="tabs__item">Как пользоваться</li>
                </ul>
            </div>
            <!-- /.tabs -->
            <div class="tabs-content">
                <h2 class="tabs-content__title">Храните деньги в банке. <br> Это надёжно и безопасно</h2>
                <p class="tabs-content__text">Вы можете быть уверены, что ваши деньги в надёжном месте, если они в&nbsp;трёхлитровой банке</p>
                <button class="button">Заказать банку</button>
            </div>
            <!-- /.tabs__content -->
        </div>
        <!-- /.container -->
    </section>

    <div class="section features">
        <div class="container">
            <h2 class="section-title section__title">Трёхлитровая банка с двойным дном</h2>
            <div class="card card-full card_bottom">
                <h3 class="card__title">часть денег проваливается на дно</h3>
                <p class="card__text">К концу месяца у вас накапливается сумма равная 10% от&nbsp;всех накоплений. Можно смело тратить их на мечту</p>
            </div>
            <!-- /.card card-full -->
            <div class="card-wrap">
                <div class="card section__card">
                    <h3 class="card__title">Деньги дома</h3>
                    <p class="card__text">Когда деньги лежат дома, у вас просто нет возможности протатить их на всякую ерунду</p>
                </div>
                <!-- /.card -->
                <div class="card section__card">
                    <h3 class="card__title">Без комиссии</h3>
                    <p class="card__text">Пополняйте банку без комисси. Доставаьть деньги можно также без комиссий и штрафов</p>
                </div>
                <!-- /.card -->
                <div class="card section__card">
                    <h3 class="card__title">Копйка рубль бережёт</h3>
                    <p class="card__text">Вы можете бросать в банку даже мелочь, которая лежала после покупок</p>
                </div>
                <!-- /.card -->
            </div>
            <!-- /.card-wrap -->
        </div>
        <!-- /.container -->
    </div>
    <!-- /.section features -->
    <section class="best">
        <div class="container">
            <h2 class="section-title">Лучшая банка в мире</h2>
            <span class="section-subtitle">По версии мамы, бабушки и тёти</span>
            <div class="card-wrap">
                <div class="card-block">
                    <div class="card best-card">
                    <img src="img/free_money.png" alt="" class="best-card__img">
                    </div>
                    <p class="best-card__text">
                        Контралируете доходы на банке есть бумажка
                    </p>
                </div>
                <!-- /.card-block -->
                <div class="card-block best__card-block">
                    <div class="card best-card">
                        <img src="img/molotok.png" alt="" class="best-card__img">
                    </div>
                    <p class="best-card__text">
                        Молоточек для самых важных случаев
                    </p>
                </div>
                <!-- /.card-block -->
                <div class="card-block best__card-block">
                    <div class="card best-card">
                        <img src="img/ruka.png" alt="" class="best-card__img best-card__img_left">
                    </div>
                    <p class="best-card__text">
                        Пополнение одним лёгким движением руки
                    </p>
                </div>
                <!-- /.card-block -->
                <div class="card-block best__card-block">
                    <div class="card best-card">
                        <img src="img/valjuta.png" alt="" class="best-card__img">
                    </div>
                    <p class="best-card__text">
                        Одна банка сколько угодно валют
                    </p>
                </div>
                <!-- /.card-block -->
            </div>
            <!-- /.card-wrap -->
        </div>
        <!-- /.container -->
    </section>
    <!-- /.best -->

    <section class="delivery">
        <div class="container">
            <h2 class="section-title">Получите банку не выходя из дома</h2>
            <form action="" class="form delivery__form">
                <div class="row">
                    <div class="col-100">
                        <input type="text" class="input form__input" placeholder="Фамилия Имя Отчество">
                    </div>
                </div>
                <!-- /.row -->
                <div class="row">
                    <div class="col-50">
                        <input type="tel" class="input form__input" placeholder="Номер телефона">
                    </div>
                    <div class="col-50">
                        <input type="email" class="input form__input" placeholder="E-mail">
                    </div>
                </div>
                <!-- /.row -->
                <div class="row">
                    <div class="col-100">
                        <input type="date" class="input form__input" placeholder="Дата рождения">
                    </div>
                </div>
                <!-- /.row -->
                <div class="button-block">
                    <button type="submit" class="button">Заказать банку</button> 
                    <!-- /.button -->
                </div>
                <!-- /.button-block -->
            </form>
        </div>
        <!-- /.container -->
    </section>
    <!-- /.delivery -->
    <footer class="footer">
        <div class="container">
            <p class="foooter__text">Название Пенькофф вымышлено и все совпрадения с реальностью случайны</p>
        </div>
    </footer>
    <script src="js/modernizr-custom.js"></script>
    <script src="js/script.js"></script>
</body>
</html>