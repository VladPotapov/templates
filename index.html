<!DOCTYPE html>
<html lang="en">
<head>
	<meta charset="UTF-8">
	<meta name="viewport" content="width=device-width, initial-scale=1.0">
	<meta http-equiv="X-UA-Compatible" content="ie=edge">
	<meta name="description" content="шаблоны для создания веб-сайтов бесплатно">
	<meta name="keywords" content="шаблоны сайтов, шаблоны сайтов html, бесплатные шаблоны сайтов, скачать шаблон сайта, готовые шаблоны сайтов, скачать шаблон сайта html, шаблоны сайтов html css, простые шаблоны сайтов, шаблоны +для создания сайта, шаблон интернет сайт, шаблоны сайтов html5, шаблоны сайтов 2018, шаблоны готовых сайтов бесплатно">
	<meta name="robots" content="All">
	<meta name="author" content="Котельников Роман">
	<meta name="yandex-verification" content="5148279a240b9c70" />
	<title>Шаблоны сайтов</title>

	<!-- Yandex.Metrika counter -->
	<script type="text/javascript" >
		(function(m,e,t,r,i,k,a){m[i]=m[i]||function(){(m[i].a=m[i].a||[]).push(arguments)};
		m[i].l=1*new Date();k=e.createElement(t),a=e.getElementsByTagName(t)[0],k.async=1,k.src=r,a.parentNode.insertBefore(k,a)})
		(window, document, "script", "https://mc.yandex.ru/metrika/tag.js", "ym");
	
		ym(52406140, "init", {
			id:52406140,
			clickmap:true,
			trackLinks:true,
			accurateTrackBounce:true
		});
	</script>
	<noscript>
		<div>
			<img src="https://mc.yandex.ru/watch/52406140" style="position:absolute; left:-9999px;" alt="" />
		</div>
	</noscript>
 <!-- /Yandex.Metrika counter -->
</head>
<body>
	<header>
		<div class="container"></div>
	</header>
	<main>
		<div class="container">Шаблоны моих работ</div>
	</main>
	<footer>
		<div class="container">
			<a href="https://webmaster.yandex.ru/sqi?host=kotelnikovroman.github.io">
				<img width="88" height="31" alt="" border="0" src="https://yandex.ru/cycounter?kotelnikovroman.github.io&theme=dark&lang=ru"/>
			</a>
		</div>
	</footer>
</body>
</html>