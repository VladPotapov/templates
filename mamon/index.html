<!DOCTYPE html>
<html lang="ru">

<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <link href="https://fonts.googleapis.com/css2?famliy=Ubuntu:wght@300;400;700&display=swap" rel="stylesheet">
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/normalize/8.0.1/normalize.min.css">
    <link rel="stylesheet" href="css/style.css">
    <title>mamon</title>
</head>

<body>
    <header>
        <div class="header">
            <a href="" class="logo">
                <span class="logo_img"></span>
                <span class="logo_text">mamon</span>
            </a>
            <!-- /.logo -->
            <nav>
                <ul class="nav">
                    <li>
                        <a href="">Services</a>
                    </li>
                    <li>
                        <a href="">Clients</a>
                    </li>
                    <li>
                        <a href="">Our time</a>
                    </li>
                    <li>
                        <button class="nav_btn" type="button">Contact Us</button>
                    </li>
                </ul>
                <!-- /.nav -->
            </nav>
        </div>
        <!-- /.header -->
    </header>
    <div class="page">
        <div class="wrap">
            <h1 class="title-main">We're an independent development agency.</h1>
            <p>
                We understand the intricacies of web development inside-out.
                That's why we build rock-solid Mobile and Web applications enriched with user-centered visual
                experiences.
            </p>
            <button class="batton" type="button">Contact Us</button>
        </div>
        <!-- /.wrap -->
    </div>
    <!-- /.page -->
    <div class="page page-top">
        <div class="wrap">
            <h2>OUR SERVICES</h2>
            <div class="services">
                <div class="services-item">
                    <img src="img/services/smartphone.svg" alt="">
                    <div>
                        <h3>Mobile Development</h3>
                        <p>
                            We develop mobile apps for all contemporary
                            mobile platforms. Mamon team builds
                            native and cross-platform solutions.
                        </p>
                    </div>
                </div>
                <!-- /.services-item -->
                <div class="services-item">
                    <img src="img/services/monitor.svg" alt="">
                    <div>
                        <h3>Web Development</h3>
                        <p>
                            Mamon offers full range of services from various
                            CMS and frameworks based websites to complex
                            ERP/CRM solutions and e-commerce stores.
                        </p>
                    </div>
                </div>
                <!-- /.services-item -->
                <div class="services-item">
                    <img src="img/services/layers.svg" alt="">
                    <div>
                        <h3>Product Design</h3>
                        <p>
                            We create design given the product audience,
                            goal, business domain, technology and
                            the platform for the solution.
                        </p>
                    </div>
                </div>
                <!-- /.services-item -->
                <div class="services-item">
                    <img src="img/services/test.svg" alt="">
                    <div>
                        <h3>Quality Assurance</h3>
                        <p>
                            Our QA team ensures high standard of
                            quality of products delivered.
                        </p>
                    </div>
                </div>
                <!-- /.services-item -->
            </div>
            <!-- /.services -->
        </div>
        <!-- /.wrap -->
    </div>
    <!-- /.page -->
</body>

</html>