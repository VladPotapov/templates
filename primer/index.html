<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <meta http-equiv="X-UA-Compatible" content="ie=edge">
    <title>Document</title>
    <link rel="stylesheet" href="css/style.css">
</head>
<body>
    <div class="box_one">
        <h1>My Auto</h1>
    </div>

    <div class="box_tue">
        <h2>I is super programmist</h2>
        Lorem ipsum dolor sit, amet consectetur adipisicing elit. Nostrum possimus blanditiis incidunt ut esse hic eveniet, harum, tempore distinctio magnam officia explicabo quas. Quam eveniet in optio ipsum, animi autem?
        Reprehenderit cumque explicabo illum, saepe et doloremque perspiciatis sunt minima tempore aut eveniet praesentium eaque, molestiae quod velit quas iusto dolorem vero nostrum repudiandae consequuntur. Qui nulla harum veritatis hic.
        Minima, perspiciatis voluptate? Voluptates assumenda animi modi eveniet neque commodi tempore nulla fuga vero ipsum enim quae consequatur beatae error, provident itaque minus praesentium rerum, hic eum totam dignissimos consectetur.
        Molestias quis aperiam aspernatur non incidunt culpa, placeat nam quidem rerum exercitationem aliquid consectetur, ad cumque cum mollitia hic vero at veritatis vel. Totam officia, id voluptas laborum labore reiciendis.
        Doloribus inventore repellat minus similique ratione nisi tempora reprehenderit error aspernatur consequuntur, quod, reiciendis dolorem obcaecati, impedit dicta iure? Delectus odit laudantium modi quo dignissimos. Ex quaerat commodi neque porro?
        Consequuntur fuga consequatur reiciendis sequi blanditiis, deserunt provident reprehenderit possimus mollitia magni dicta quam illo delectus tempore doloribus. Aspernatur minima facilis odit aperiam quia magni nisi, aliquam debitis quae veritatis.
        Reprehenderit id ullam est ut at eligendi illum tenetur mollitia sunt dolorem quo dolores enim consequuntur dolor voluptatem cum quisquam, sint officia. Vel voluptate, fugit nemo dicta corporis accusamus deleniti?
        <table>
            <tr>
                <td>Eligendi pariatur quia quam hic sit sunt libero molestiae vitae ipsa rerum </td>
                <td>quibusdam tempore fugit odit amet, cumque facilis numquam, voluptatum </td>
                <td>consequatur dolore at eaque laborum. Consequuntur odit commodi eum.</td>
            </tr>
            <tr>
                <td>Repellendus, odit ex culpa, eum minus eius aspernatur, dolor soluta nisi rem itaque at voluptates. Alias beatae possimus perspiciatis. Beatae pariatur unde quas tenetur, necessitatibus repellat amet adipisci libero blanditiis.</td>
            </tr>
            <tr>
                <td>Nam qui at consectetur obcaecati dolor hic excepturi nostrum voluptates dolorem dolorum quidem libero labore magnam atque, deleniti </td>
                <td>incidunt officia nesciunt numquam veritatis quisquam fugiat omnis cumque! Suscipit, veritatis voluptatem?</td>
            </tr>
        </table>
    </div>
    <div class="basic basic_one">
        <div class="basic basic_tue">block children</div>
    </div>
    <div class="transform">
        <div class="transform_one"></div>
        <div class="transform_two"></div>
        <div class="transform_three"></div>
    </div>
    <textarea name="" id="" cols="30" rows="10"></textarea>
    <div class="floats">
        <div class="right"></div>
    </div>
</body>
</html>