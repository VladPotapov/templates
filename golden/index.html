<!DOCTYPE html>
<html lang="en">
<head>
	<meta charset="UTF-8">
	<meta name="viewport" content="width=device-width, initial-scale=1.0">
	<meta http-equiv="X-UA-Compatible" content="ie=edge">
	<meta name="description" content="адаптивный шаблон сайта для компании golden">
	<meta name="keywords" content="скачать шаблон сайта, шаблон сайта скачать бесплатно, скачать шаблон сайта html, сайт визитка шаблон, одностраничный сайт шаблон, адаптивный шаблон сайта, шаблон интернет сайт, шаблон сайта компании, html шаблон, адаптивный html шаблон golden">
	<meta name="robots" content="All">
	<meta name="author" content="Котельников Роман">
	<meta name="yandex-verification" content="5148279a240b9c70" />
	<title>Golden</title>
	<link rel="stylesheet" href="../../css/style.css">
	<link rel="stylesheet" href="css/style.css">
	<link rel="stylesheet" href="css/adaptivity.css" media="only screen and (max-width: 929px)">

	<!-- Yandex.Metrika counter -->
	<script type="text/javascript" >
		(function(m,e,t,r,i,k,a){m[i]=m[i]||function(){(m[i].a=m[i].a||[]).push(arguments)};
		m[i].l=1*new Date();k=e.createElement(t),a=e.getElementsByTagName(t)[0],k.async=1,k.src=r,a.parentNode.insertBefore(k,a)})
		(window, document, "script", "https://mc.yandex.ru/metrika/tag.js", "ym");
	
		ym(52406140, "init", {
			id:52406140,
			clickmap:true,
			trackLinks:true,
			accurateTrackBounce:true
		});
	</script>
	<noscript>
		<div>
			<img src="https://mc.yandex.ru/watch/52406140" style="position:absolute; left:-9999px;" alt="" />
		</div>
	</noscript>
 <!-- /Yandex.Metrika counter -->
</head>
<body>
	<header>
		<div class="container">
			<div class="heading clearfix">
				<img src="img/logo.png" alt="Golden" class="logo">
				<div class="button_menu">
					<a href="#" class="button_menu_link">
						<span></span>
					</a>
				</div>
				<nav>
					<ul class="menu">
						<li><a href="">home</a></li>
						<li><a href="">Services</a></li>
						<li><a href="">Portfolio</a></li>
						<li><a href="">About</a></li>
						<li><a href="">Contact</a></li>
					</ul>
				</nav>
			</div><!-- heading -->
			<div class="titles">
				<div class="title__first">
					Welcome To Our Studio!
				</div>
				<h1>It’s nice to Meet you</h1>
			</div><!-- titles -->
			<a href="" class="button">Tell Me More</a>
		</div>
	</header>
	<section id="services">
		<div class="container">
			<div class="title">
				<h2>Services</h2>
				<p>Proin iaculis purus consequat sem cure</p>
			</div>
			<div class="services clearfix">
				<div class="services_item">
					<img src="img/icon1.png" alt="">
					<h3>E-Commerce</h3>
					<p>Proin iaculis purus consequat sem cure 
					digni ssim. Donec porttitora entum suscipit 
					aenean rhoncus posuere odio in tincidunt.</p>
				</div>
				<div class="services_item">
					<img src="img/icon2.png" alt="">
					<h3>Responsive Web</h3>
					<p>Proin iaculis purus consequat sem cure 
					digni ssim. Donec porttitora entum suscipit 
					aenean rhoncus posuere odio in tincidunt.</p>
				</div>
				<div class="services_item">
					<img src="img/icon3.png" alt="">
					<h3>Web Security</h3>
					<p>Proin iaculis purus consequat sem cure 
					digni ssim. Donec porttitora entum suscipit 
					aenean rhoncus posuere odio in tincidunt.</p>
				</div>
			</div>
		</div>
	</section>
	<section id="portfolio">
		<div class="container">
			<div class="title">
				<h2>Our portfolio</h2>
				<p>Proin iaculis purus consequat sem cure</p>
			</div>
			<div class="works clearfix">
				<img src="img/img1.jpg" alt="">
				<img src="img/img2.jpg" alt="">
				<img src="img/img3.jpg" alt="">
			</div>
		</div>
	</section>
	<section id="about">
		<div class="container">
			<div class="title">
				<h2>about us</h2>
				<p>Proin iaculis purus consequat sem cure</p>
			</div>
			<div class="about_article clearfix">
				<div class="left">
					<div class="about_img">
						<img src="img/about1.jpg" alt="">
					</div>
					<div class="about_content">
						<h2>July 2010</h2>
						<h3>Our Humble Beginnings</h3>
						<p>Proin iaculis purus consequat sem cure 
						digni ssim. Donec porttitora entum suscipit 
						aenean rhoncus posuere odio in tincidunt. Proin 
						iaculis purus consequat sem cure digni 
						ssim. Donec porttitora entum suscipit.</p>
					</div>
				</div>
			</div><!-- end about_article -->
			<div class="about_article clearfix">
				<div class="right">
					<div class="about_img">
						<img src="img/about2.jpg" alt="">
					</div>
					<div class="about_content">
						<h2>January 2011</h2>
						<h3>Facing Startup Battles</h3>
						<p>Proin iaculis purus consequat sem cure 
						digni ssim. Donec porttitora entum suscipit 
						aenean rhoncus posuere odio in tincidunt. Proin 
						iaculis purus consequat sem cure digni 
						ssim. Donec porttitora entum suscipit.</p>
					</div>
				</div>
			</div><!-- end about_article -->
			<div class="about_article clearfix">
				<div class="left">
					<div class="about_img">
						<img src="img/about3.jpg" alt="">
					</div>
					<div class="about_content">
						<h2>December 2012</h2>
						<h3>Enter The Dark Days</h3>
						<p>Proin iaculis purus consequat sem cure 
						digni ssim. Donec porttitora entum suscipit 
						aenean rhoncus posuere odio in tincidunt. Proin 
						iaculis purus consequat sem cure digni 
						ssim. Donec porttitora entum suscipit.</p>
					</div>
				</div>
			</div><!-- end about_article -->
			<div class="about_article clearfix">
				<div class="right">
					<div class="about_img">
						<img src="img/about4.jpg" alt="">
					</div>
					<div class="about_content">
						<h2>february 2014</h2>
						<h3>Our Triumph</h3>
						<p>Proin iaculis purus consequat sem cure 
						digni ssim. Donec porttitora entum suscipit 
						aenean rhoncus posuere odio in tincidunt. Proin 
						iaculis purus consequat sem cure digni 
						ssim. Donec porttitora entum suscipit.</p>
					</div>
				</div>
			</div><!-- end about_article -->
		</div><!-- end container -->
	</section>
	<footer>
		<div class="container">
				<a href="https://webmaster.yandex.ru/sqi?host=kotelnikovroman.github.io">
					<img width="88" height="31" alt="" border="0" src="https://yandex.ru/cycounter?kotelnikovroman.github.io&theme=dark&lang=ru"/>
				</a>
		</div>
		<div class="container">
			<p>© Copyright 2014 FreebiesXpress.com</p>
		</div>
	</footer>
	<div class="donwload_zip">
		<a href="golden.zip" title="download">
				<img src="../../img/downloads.png" alt="">
		</a>
	</div>
</body>
<script src="js/jquery-3.3.1.js"></script>
<script src="js/script.js"></script>
</html>